styles/layouts/Error.module.scss

```diff
@@ -0,0 +1,24 @@
+.container {
+  @apply flex flex-col;
+  @apply size-full h-screen;
+}
+
+.page {
+  @apply flex flex-col flex-1;
+  @apply p-4 gap-8;
+  @apply size-full;
+  @apply justify-center items-center text-center;
+  @apply bg-color-1;
+
+  h1, p {
+    @apply text-white;
+  }
+
+  h1 {
+    @apply text-6xl font-bold;
+  }
+
+  .button{
+    @apply m-2;
+  }
+}
```

Comment:
it might be slightly better to do 
```scss
.container {
  @apply flex flex-col;
  @apply size-full h-screen;
  @apply bg-color-3;

  .page {
    @apply flex flex-col flex-1;
    @apply p-4 gap-8;
    @apply size-full;
    @apply justify-center items-center text-center;
    @apply bg-color-1;
    @apply rounded-b-2xl;

    h1, p {
      @apply text-white;
    }

    h1 {
      @apply text-6xl font-bold;
    }

    .button{
      @apply m-2;
    }
  }
}
```
instead so that its nested more but that's a small nit